src/lib/exportUtils.ts

```
@@ -0,0 +1,101 @@
+    try {
+        // Создаем копию контента
+        const exportContent = content.cloneNode(true) as HTMLElement;
+        
+        // Получаем все стили
+        const styles = Array.from(document.styleSheets)
+            .map(styleSheet => {
+                try {
+                    return Array.from(styleSheet.cssRules)
+                        .map(rule => rule.cssText)
+                        .join('\n');
+                } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)